### PR TITLE
Pagination widget: save items per page in cookie

### PR DESCRIPTION
**Pull request by orangejenny**
Largely motivated by the ICDS team, who spend a surprising amount of time going through the first few pages of builds looking for the latest build in a specific stream.

Review by commit.

@millerdev / @proteusvacuum

### CODE REVIEW

**Review by proteusvacuum — APPROVED** [review]
:cookie: 
This is great!

**Review by millerdev — APPROVED** [review]

File: corehq/apps/app_manager/views/releases.py [hunk 1]
```diff
@@ -186,7 +185,6 @@ def get_releases_context(request, domain, app_id):
         'build_profile_access': build_profile_access,
         'application_profile_url': reverse(LanguageProfilesView.urlname, args=[domain, app_id]),
         'lastest_j2me_enabled_build': CommCareBuildConfig.latest_j2me_enabled_config().label,
-        'fetchLimit': request.GET.get('limit', DEFAULT_FETCH_LIMIT),
```
Comment (esoergel) [inline corehq/apps/app_manager/views/releases.py]: Wow, that's a lot of :x:, nice!

File: corehq/apps/export/templates/export/partials/export_list_controller.html [hunk 2]
```diff
@@ -44,6 +44,7 @@
                 <pagination data-apply-bindings="false"
                             data-bind="visible: showPagination"
                             params="goToPage: goToPage,
+                                    slug: 'export-list',
```
Comment (esoergel) [inline corehq/apps/export/templates/export/partials/export_list_controller.html]: this is fantastic!  Love the small footprint.  What happens if `slug` isn't set?  Just that it won't save the page limit?

**Review by esoergel — APPROVED** [review]
Love it.  My only concern is that if you give ICDS a cookie, they'll want a glass of milk.
:mouse: :cookie:

File: corehq/apps/export/templates/export/partials/export_list_controller.html (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/export/templates/export/partials/export_list_controller.html]: yep

**Pull request merged**